lib/puppet/version.rb

```diff
@@ -8,7 +8,7 @@
 # Raketasks and such to set the version based on the output of `git describe`
 
 module Puppet
-  PUPPETVERSION = '8.8.0'
+  PUPPETVERSION = '8.8.1'
```

Comment:
I think we want to keep version `8.9.0` here and below in preparation for the next release.